rpc/cloudagent/security/role/moc_cloudagent_role.proto

```diff
@@ -28,9 +37,13 @@ message RoleResponse {
 }
 
 message Action {
-	AccessOperation operation = 1;
+	AccessOperation operation = 1 [deprecated=true];
 	// ProviderType the rule works on
 	ProviderType providerType = 2;
+	// General access permissions
+	GeneralAccessOperation newOperation = 3;
```

Comment:
could we call this generalOperation instead of newOperation?

Reply:
Resolved